src/app/login/login.component.ts

```diff
@@ -32,6 +33,7 @@ import { LoginService } from './login.service';
 export class LoginComponent {
   faGithub = faGithub;
   faGoogle = faGoogle;
+  public state: NumberInput;
```

Comment:
https://material.angular.io/components/tabs/api selectedInex says the type is `number`, just use that. Not familiar with cdk/coercion.